### ISSUE TITLE
Can't upgrade to 0.3.9 due to dependency conflict

### ISSUE THREAD

**Issue by miklcct** [matanshukry/flutter_google_places_sdk#99]
```
flutter pub add flutter_google_places_sdk:^0.3.9
"flutter_google_places_sdk" is already in "dependencies". Will try to update the constraint.
Because flutter_google_places_sdk >=0.3.9 depends on flutter_google_places_sdk_web ^0.1.7 which depends on google_maps ^7.1.0, flutter_google_places_sdk >=0.3.9 requires
  google_maps ^7.1.0.
And because google_maps >=7.1.0 <8.0.0-dev.3 depends on web ^0.5.0 and share_plus >=10.0.1 depends on web ^1.0.0, flutter_google_places_sdk >=0.3.9 is incompatible with share_plus
  >=10.0.1.
So, because aubin_mol_challenge depends on both share_plus ^10.0.2 and flutter_google_places_sdk ^0.3.9, version solving failed.
```

You have introduced a new dependency from 0.3.8 (the version I currently using) to 0.3.9, with the previous version using google_maps 6.3.0 which doesn't have a hard dependency on web, therefore I have failed to upgrade to 0.3.9

**Comment by matanshukry** [comment]
This should be solved by upgrading `google_maps` to version `8.0.0`, which allows a range of versions on `web`, including the one mentioned here - `1.0.0`

**Comment by matanshukry** [comment]
So I actually already got a branch to update it, but it's failing due to an issue with google_maps dependency: https://github.com/a14n/dart-google-maps/issues/138

Until then we can't upgrade

**Comment by SzymonDziak** [comment]
Same error here. when is this gonna be fixed?

**Comment by matanshukry** [comment]
@SzymonDziak it's waiting on the dependency, as I wrote above.
I can see there's already a PR with a fix in the dependent repo - https://github.com/a14n/dart-google-maps/pull/140 - but need to wait for it to merge before we can fix it

**Comment by TatsuUkraine** [comment]
@matanshukry it seems fix got merged and new version is available

**Comment by matanshukry** [comment]
Updated the version in the PR, merged, and published a new version (^0.1.8).

Closing as this should be fixed now - but please check and if there is still an issue feel free to reopen.
